ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/site/js/accordionview.js

```diff
@@ -604,6 +607,18 @@
         #convertToPanelId(idToConvert) {
             return idToConvert.substring(0, idToConvert.lastIndexOf("-")) + Accordion.idSuffixes.panel;
         }
+        getRepeatableRootElement(childView){
+          return childView.element.parentElement.parentElement.parentElement;
```

Comment:
there seems a strong coupling on the dom structure here. we should traverse the tree to find the repeatable element rather than using parent Element n number of times.

Reply:
Changed